synapse/cores/common.py

```diff
@@ -228,34 +228,34 @@ def __init__(self, link, store, **conf):
 
         s_ingest.IngestApi.__init__(self, self)
 
-    def addRuntNode(self, form, valu, **props):
+    def addRuntNode(self, rform, valu, **props):
```

Comment:
why were all these changed to rform?

Reply:
'form' is a possible value for a prop passed to a runt node when making a syn:prop node.